### PR TITLE
Converts MO19 from gateway to ruin

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Converts the MO19 gateway mission to a space ruin. Mapping was initially done by @S34NW until I stole it and shrunk it a bit.

The station part of the map is the same base template, but with xenos moved into the main part of the map as opposed to a hive on the top left. The escape room on the top right has also been removed due to being entirely superfluous. 

The asteroid is also dug out on the bottom to provide access to the external airlocks on the aft side.

## Why It's Good For The Game
Gateway bad, ruins good.

## Images of changes
See MDB or MDB2. Sidenote, I need to kill MDB1 soon.

## Testing
image
Seems good

## Changelog
:cl: AffectedArc07
tweak: The MO19 gateway is now a random space ruin instead of a gateway destination.
/:cl:

### CODE REVIEW

**Review by S34NW — CHANGES_REQUESTED** [review]
image
No `/mob/living/simple_animal/hostile/alien/queen/large`?

image
You should probably put some decoration outside the windows like here, there are 6 such windows areas. Not sure what you could put there but it would look better than just sand.

image
Your air vents are the wrong subtype and var edited to be on.

image
image
image
image
Consoles are wrong dir

image
This injector is var edited to be on instead of subtype

image
These stools are all lacking dir

image
Some of the floors in this hallway are properly airless, but others are not.

**Comment by AffectedArc07** [comment]
<details>
<summary>Sean's comment</summary>

> image No `/mob/living/simple_animal/hostile/alien/queen/large`?
> 
> image You should probably put some decoration outside the windows like here, there are 6 such windows areas. Not sure what you could put there but it would look better than just sand.
> 
> image Your air vents are the wrong subtype and var edited to be on.
> 
> image image image image Consoles are wrong dir
> 
> image This injector is var edited to be on instead of subtype
> 
> image These stools are all lacking dir
> 
> image Some of the floors in this hallway are properly airless, but others are not.

</details>

All of these are things you could have fixed when you did the map on your own PR :smork:

**Comment by S34NW** [comment]
> All of these are things you could have fixed when you did the map on your own PR :smork:

You maade me close it prematurely, so I blame you :P

**Comment by Komrad822** [comment]
Will the stechkin still be there next to the queen or no?

**Comment by AffectedArc07** [comment]
> Will the stechkin still be there next to the queen or no?

Nope

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -301,3 +301,11 @@
 	description = "An old transport ship, possibly with a dubious past. It smells faintly of meat."
 	allow_duplicates = FALSE
 	cost = 2 // Pretty big
+
+/datum/map_template/ruin/space/mo19
+	id = "mo19"
+	suffix = "moonoutpost19.dmm"
+	name = "Moon Outpost 19"
+	description = "A now-defunct lunar outpost, with the last received signal being that of distress."
```
Comment (hal9000PR) [inline code/datums/ruins/space.dm:309]: its not longer on a moon, its an asteroid.

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**